ios/FluentUI/Avatar/AvatarView.swift

```diff
@@ -258,6 +258,12 @@ open class AvatarView: UIView {
         }
     }
 
+    @objc open var shouldCalculateRingColor: Bool = false {
```

Comment:
can you add documentation on this public variable please?

Reply:
instead of `shouldCalculateRingColor`, should we say something like `shouldGenerateBorderColor` or something like that because ring wouldn't make sense based on other AvatarView's public properties?

Reply:
Yes makes sense